From 24a3b87317a893dd8b88f30be97d0a36a93a596c Mon Sep 17 00:00:00 2001
From: Adam Jackson <ajax@redhat.com>
Date: Fri, 18 Jan 2013 13:01:12 -0500
Subject: [PATCH 1/2] i2c: Don't scream on literally every single write to SCL
 or SDA

Signed-off-by: Adam Jackson <ajax@redhat.com>
---
 src/vb_i2c.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/vb_i2c.c b/src/vb_i2c.c
index c76bd4d..d238d90 100755
--- a/src/vb_i2c.c
+++ b/src/vb_i2c.c
@@ -1606,8 +1606,8 @@ VOID vWriteClockLine(PXGI_HW_DEVICE_INFO pHWDE, UCHAR data)
 		vWriteClockLineDVI(pHWDE, data);
 	else if(pHWDE->crtno == 2)
 		vWriteClockLineFCNT(pHWDE, data);
-
-	ErrorF("Error(XGI) : Unknown output device!\n");
+	else
+	    ErrorF("Error(XGI) : Unknown output device!\n");
 }
 
 VOID vWriteDataLine(PXGI_HW_DEVICE_INFO pHWDE, UCHAR data)
@@ -1618,8 +1618,8 @@ VOID vWriteDataLine(PXGI_HW_DEVICE_INFO pHWDE, UCHAR data)
 	   vWriteDataLineDVI(pHWDE, data);
 	else if(pHWDE->crtno == 2)
 		vWriteDataLineFCNT(pHWDE, data);
-
-	ErrorF("Error(XGI) : Unknown output device!\n");
+	else
+	    ErrorF("Error(XGI) : Unknown output device!\n");
 }
 
 BOOLEAN bReadClockLine(PXGI_HW_DEVICE_INFO pHWDE)
-- 
1.8.0.2

